lib/doctrine.js

```diff
@@ -526,6 +526,17 @@
             return true;
         };
 
+        TagParser.prototype.parseCaption = function parseDescription() {
+            var description = trim(sliceSource(source, index, this._last));
+            if (description.match(/^\s*<caption>([\s\S]+?)<\/caption>(\s*[\n\r])([\s\S]+)$/i)) {
+                this._tag.caption = RegExp.$1;
```

Comment:
A bit too much regex magic for my taste. Can you do this as just locating the tags and using slice() or substring() to get this info?

Also, please don't use the dollar properties on RegExp, these are deprecated in JS and aren't clear on what they refer to.